core/nginx-ingress/release.yaml

```diff
@@ -44,4 +44,11 @@ spec:
                 values:
                   - 'true'
       nodeSelector:
-        kubernetes.io/os: "linux"
\ No newline at end of file
+        kubernetes.io/os: "linux"
+      config:
+        entries:
+          http2: "True"
+      podDisruptionBudget:
+        enabled: true
+        annotations: {}
+        minAvailable: 1
```

Comment:
prob should be 2 min available, with 3 in production

Reply:
Moved to the environment values patch, and set to 2 for the SND1.